### PR TITLE
Added function to create discrete geometry from level set vector

### PR DESCRIPTION
**Pull request by ConnorMallon**
Added function to create discrete geometry from level set vector

### CODE REVIEW

File: src/LevelSetCutters/DiscreteGeometries.jl [hunk 1]
```diff
@@ -65,3 +65,9 @@ function _find_unique_leaves(tree)
   j_to_fun, oid_to_j
 end
 
+function discrete_geometry(
```
Comment (fverdugo) [inline src/LevelSetCutters/DiscreteGeometries.jl:68]: Can you change the name of this function to `DiscreteGeometry`? 
it is just a constructor of a `DiscreteGeometry` object. This is why we want it in camel case.

File: src/LevelSetCutters/LevelSetCutters.jl [hunk 2]
```diff
@@ -35,6 +35,7 @@ export cylinder
 export plane
 export cube
 export discretize
+export discrete_geometry
```
Comment (fverdugo) [inline src/LevelSetCutters/LevelSetCutters.jl:38]: export `DiscreteGeometry` instead

**Comment by ConnorMallon** [comment]
I have added a test in the DiscreteGeometriesTests.jl file. If I am correct, this just checks that the Discrete geometry is a valid "geometry" rather than actually seeing if it is going to give you a correct answer when solving a problem (which would instead be a test like the modified poissonCutFEM test). Is what I have done here sufficient for testing?

**Comment by fverdugo** [comment]
> I have added a test in the DiscreteGeometriesTests.jl file. If I am correct, this just checks that the Discrete geometry is a valid "geometry" rather than actually seeing if it is going to give you a correct answer when solving a problem (which would instead be a test like the modified poissonCutFEM test). Is what I have done here sufficient for testing?

This is a good question. The tests should (at least) execute the code you want to test. Ideally, they should also check that the result is numerically correct. However, it is not always straightforward to do. So, in many situations, we only execute the code, and perhaps we check once visually if the results make sense. But, as I have said, the ideal thing is to check results numerically.

For the current case, I think it is enough what you have done.

**Pull request merged**